### PR TITLE
Updates project name

### PR DESCRIPTION
**Pull request by caseyyee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# Old Fashioned
-A Next.js starter with scss, autoprefixer, SSR, and static generator
+# Mozilla VR site
+
+Mozilla VR site, built using Next.js starter with scss, autoprefixer, SSR, and static generator
```
Comment (cvan) [inline README.md]: not: I'd like to next.js, SCSS

**Comment by slightlyoffbeat** [comment]
Can you update to not include the package-lock.json?

**Comment by cvan** [comment]
@slightlyoffbeat can you give me and @caseyyee access perms to this repo? I can then edit his PRs

**Comment by slightlyoffbeat** [comment]
@cvan I'm going to start transferring this repo to the mozvr team so we can pick up there where you'll have access/permissions etc.

**Comment by caseyyee** [comment]
merging, will address package-lock.json in another commit.

**Pull request merged**